### PR TITLE
Campus partners page

### PR DESCRIPTION
**Pull request by jayp822**
#### also need the links for the orgs

demo11

### CODE REVIEW

**Comment by justinquinnb** [comment]
Will send shortly, thank you!

**Comment by justinquinnb** [comment]
Just noticed, affiliates is missing an extra F!

**Comment by justinquinnb** [comment]
ACM: https://acm-uga.github.io/
CSCI: https://uga.campuslabs.com/engage/organization/csci
SOE: https://www.soeuga.com/
KTP: https://ktpgeorgia.com/

I will have to make card images for these clubs tonight, which I'll add here, since we don't have an asset library from them.

**Comment by justinquinnb** [comment]
Here are the values for the card fields. Check out the Figma, we've since added Hacks and now have an official GDGC wordmark you need to change out.

HacksX Card Image
**Club Name:** UGA Hacks
**Club Description:** Every spring, UGA Hacks puts on a 3-day, 800+ person hackathon at MLC.

SOE Card Image
**Club Name:** Society of Entrepreneurs
**Club Description:** Learn from and connect with founders and business-savvy individuals at SOE's many events.

ACM Card Image
**Club Name:** ACM @ UGA
**Club Description:** Every week, ACM hosts workshops discussing the latest technologies.

CSCI Card Image
**Club Name:** Computer Science Career Initiative
**Club Description:** CSCI provides resources to assist students in their pursuit of a many technical fields.

KTP Card Image
**Club Name:** Kappa Theta Pi: Phi Colony
**Club Description:** KTP is UGA's first technology frat, developing the next generation of industry professionals.

GDGC Card Image
**Club Name:** Google Developer Groups on Campus: UGA
**Club Description:** GDGC UGA co-hosts their meetings with DevDogs, getting you in the Google sphere.

**Comment by jayp822** [comment]
https://github.com/user-attachments/assets/33e9b821-52b3-4bdb-8961-47dd1a33d0e1

**Comment by justinquinnb** [comment]
Give me a ping when you finish the final touches. Since my last comment, we also finally got a GDGC page going that you can add to the card if possible: https://gdg.community.dev/gdg-on-campus-university-of-georgia-athens-united-states/
image

File: src/app/sponsor/our-partners/page.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        "Every spring, UGA Hacks puts on a 3-day, 800+ person hackathon at MLC.",
+      link: "https://ugahacks.com/",
+    },
+  ];
+  return (
+    <div className="w-full section ">
+      <PageTitleTemplate
+        redText="Partners"
+        blackText="Campus "
+        reverse="true"
```
Comment (nivedhanatarajan7) [inline src/app/sponsor/our-partners/page.js:69]: I think the space between the title and the next line can be shorter.

Web View:
image

Mobile View:
image

File: src/app/sponsor/our-partners/page.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+      <div className="text-center text-[2rem] sm:text-[2.5rem] mt-[4rem]">
+        <h2 className="font-[700] inline  text-BulldogRed">Campus </h2>
+        <h2 className="font-[700] inline text-MidnightBlue">Affiliates</h2>
+      </div>
+      <EmblaCarousel
+        slides={imagesForCarousel}
+        banner={true}
+        options={OPTIONS}
+      />
+
```
Comment (nivedhanatarajan7) [inline src/app/sponsor/our-partners/page.js:101]: The space between the carousel and the campus partners' images can be shorter in mobile view.

Mobile view:
image

**Review by nivedhanatarajan7 — CHANGES_REQUESTED** [review]
Hi! The campus partners page looks really great! I gave some comments on spacing in the page. Thank you!

**Review by nivedhanatarajan7 — APPROVED** [review]

**Pull request merged**